charts/gateway/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: "3.3.0"
+appVersion: "3.3.1"
 description: Conduktor Gateway chart
 name: conduktor-gateway
-version: 3.3.0
+version: 3.3.1
```

Comment:
Do not increment the chrat version here, it will be done on create release workflow